internal/armada/configuration/types.go

```diff
@@ -248,7 +252,7 @@ type JetstreamConfig struct {
 
 type QueueManagementConfig struct {
 	AutoCreateQueues       bool
-	DefaultPriorityFactor  queue.PriorityFactor
```

Comment:
I changed this because it was causing a circular reference.  Essentially other armada module imported by the `configuration` package is problematic because configuration is imported everywhere.  In this case all we were using this for was a type alias so it shouldn't really matter.